### PR TITLE
Feature/class endpoints

### PR DESCRIPTION
**Pull request by F-Adam-B**
Here are the tests, and initial router endpoints for the class. The router endpoints are just skeletons since I didn't know how to test them without a database and schema created. Will update as we lay out the db schema structure.

### CODE REVIEW

**Review by visaudi — CHANGES_REQUESTED** [review]
Let's get .nyc out

File: class/router.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+router.get('/', (req, res) => {
+    res.json([{'classname': 'BIO103'}]);
+  });
+
+
+router.post('/', jsonParser, (req, res) => {
+   
+    const item = {'id':'123', 'firstName':'John', 'lastName':'Doe', 'username':'username', 'className':'BIO103'}
```
Comment (visaudi) [inline class/router.js]: Can this be the multiline object form? All on one line is hard to read for 14.

**Review by visaudi — CHANGES_REQUESTED** [review]
Just make the mentioned object multiline (and well formatted).

File: class/router.js (hunk 1 above)
Comment (F-Adam-B) [reply class/router.js]: Okay, should be all set. If cool, I'm going to continue to build out the endpoints to accept basic data from the db.

**Pull request merged**